src/android/src/main/java/com/reactnativecommunity/slider/ReactSlider.java

```diff
@@ -233,4 +234,23 @@ public void setThumbImage(final String uri) {
       setThumb(getThumb());
     }
   }
+    @Override
+    public boolean onTouchEvent(MotionEvent event) {
+        //设置父容器不拦截事件
+        if (getParent() != null) {
+            try {
```

Comment:
What exceptions do we expect here to be thrown by `MotionEvent`?
`getParent()` is already checked against `null`.